models/release.go

```diff
@@ -469,6 +470,24 @@ func DeleteReleaseByID(id int64, u *User, delTag bool) error {
 		go HookQueue.Add(rel.Repo.ID)
 	}
 
+	if setting.AttachmentEnabled {
```

Comment:
This line is unnecessary. Considering we change the app.ini after uploaded some attachments.